examples/deployment/catalog_item/main.tf

```diff
@@ -27,7 +27,49 @@ resource "vra_deployment" "this" {
     count    = 1
     flag     = false
   }
+}
+
+output "resources" {
+  description = "All the resources from a vRA deployment"
+  value = vra_deployment.this.resources
+}
+
+output "resource_properties_by_name" {
+  description = "Properties of all resources by its name from a vRA deployment"
+  value = {
```

Comment:
The added examples in this file could use some additional indentation to improve readability.

Reply:
Done.